2019/python/rules_engine.py

```diff
@@ -1055,10 +1055,10 @@ def v644_1(self):
 	def v644_2(self):
 		"""An invalid Sex data field was reported.
 		2) If Sex of Applicant or Borrower equals 6, then Sex of Applicant or Borrower Collected on the Basis of
-		Visual Observation or Surname must equal 2."""
+		Visual Observation or Surname must equal 2 or 3."""
 		field = "Applicant Sex Basis"
 		edit_name = "v644_2"
-		fail_df = self.lar_df[(self.lar_df.app_sex=="6")&(self.lar_df.app_sex_basis!="2")]
+		fail_df = self.lar_df[(self.lar_df.app_sex=="6")&(~self.lar_df.app_sex_basis.isin(("2", "3")))]
```

Comment:
What is the source of including 3? I can't find it in 2018 or 2019 FIG under v644

Reply:
This change is reflected in the last commit.